src/setupWorker/glossary.ts

```diff
@@ -53,7 +53,9 @@ export type StartOptions = SharedOptions & {
    * Disable the logging of captured requests
    * into browser's console.
    */
-  quiet?: boolean
+  quiet?:
+    | boolean
+    | ((req?: MockedRequest, res?: ResponseWithSerializedHeaders) => boolean)
```

Comment:
By the time we analyze the `quiet` option and output a successful mock to console both `req` and `res` exist. No need to annotate them as optional with `?`.

Reply:
this was to handle `src/setupWorker/start/utils/activateMocking.ts`, there quiet can be a function without and req and res. Or am I missing anything?

Reply:
Yeah, I understand the intention. As I've stated earlier, such internal events like `ACTIVATE_MOCKING` shouldn't be controlled by the predicate function of the `quiet` option.

Reply:
Should we remove the quiet check from this log altogether?

Reply:
That's a sensible approach, yet it would be a breaking change then. Previously this log was also controlled by `options.quiet`. 

Alternatively, maybe we should distribute a function-as-value in a _different_ option?

Reply:
yeah, it makes sense to introduce different option to handle this case.
but then usage seems a bit tricky,
```
{
quiet: true,
quietPredicate: (req) => {
      return req.url.href.includes('profile')
  },
}
```
what is expected might be a not sure.

we could ntroducte another option  for internal logs
```
quiet:  (req) => {
      return req.url.href.includes('profile')
  },
quietInternals: true
```
we can come up with better names for sure 😅